assets/scss/dropdown.scss

```diff
@@ -136,15 +138,15 @@
             padding: 0.25rem 1rem;
           }
 
-          input[type="checkbox"] {
+          input[type="checkbox"],input[type="radio"] {
             left: 0;
             opacity: 0;
             position: fixed;
             top: 0;
             z-index: -10;
           }
 
-          input[type="checkbox"]:checked + label { 
+          input[type="checkbox"]:checked + label,input[type="radio"]:checked + label {
```

Comment:
See previous comment, move `input[type="radio"]:checked + label` to the next line.